tests/performance/uniq.xml

```diff
@@ -23,7 +23,6 @@
                 -->
                <value>SearchEngineID</value>
                <value>RegionID</value>
-               <value>SearchPhrase</value>
```

Comment:
It has removed SearchPhrase key from all aggregate functions (uniq, uniqExact, uniqCombined, etc).